src/server/services/parkeren/parkeren.ts

```diff
@@ -1,50 +1,87 @@
+import FormData from 'form-data';
+
 import { FeatureToggle } from '../../../universal/config/feature-toggles';
 import { apiSuccessResult } from '../../../universal/helpers/api';
+import { isMokum } from '../../../universal/helpers/brp';
 import { AuthProfileAndToken } from '../../auth/auth-types';
-import { DataRequestConfig } from '../../config/source-api';
 import { getFromEnv } from '../../helpers/env';
 import { getApiConfig } from '../../helpers/source-api-helpers';
 import { requestData } from '../../helpers/source-api-request';
+import { fetchBRP } from '../brp';
 
-type ParkerenUrlTransformedResponse = {
-  isKnown: boolean;
-  url: string | null;
-};
+export async function fetchParkeren(
+  requestID: RequestID,
+  authProfileAndToken: AuthProfileAndToken
+) {
+  const brpData = await fetchBRP(requestID, authProfileAndToken);
+  const livesInAmsterdam = isMokum(brpData?.content);
+  const isCommercial = authProfileAndToken.profile.profileType === 'commercial';
+
+  let isKnown = false;
+  if (!FeatureToggle.parkerenCheckForProductAndPermitsActive) {
+    isKnown = true;
+  } else if (isCommercial) {
+    isKnown = true;
+  } else if (livesInAmsterdam) {
+    isKnown = true;
+  } else {
+    isKnown = await hasPermitsOrPermitRequests(requestID, authProfileAndToken);
+  }
```

Comment:
Je kunt er ook iets heel expliciets van maken zonder if else bloat :D
```suggestion
const checkForPossibleThemaParkeren = !FeatureToggle.parkerenCheckForProductAndPermitsActive; 
const alwaysHasThemaParkeren = isCommercial || livesInAmsterdam;

if (alwaysHasThemaParkeren || !checkForPossibleThemaParkeren) {
  isKnown = true;
} else {
  isKnown = await hasPermitsOrPermitRequests(requestID, authProfileAndToken);
}
```

Reply:
Of
```
const shouldCheckForPermitsOrPermitRequests =
  authProfileAndToken.profile.profileType === 'private' &&
  !livesInAmsterdam &&
  FeatureToggle.parkerenCheckForProductAndPermitsActive; 

let isKnown = true;

if (shouldCheckForPermitsOrPermitRequests) {
  isKnown = await hasPermitsOrPermitRequests(requestID, authProfileAndToken);
}
```

Reply:
Ah ja nice had al zoiets gedaan, alleen ietsje anders maar dit is ook ietsje beter. Heb alleen de `not`s omgedraaid zodat het wat makkelijker te lezen is